app.js

```diff
@@ -1,8 +1,122 @@
 // import functions and grab DOM elements
 
+const cup1ImageEl = document.querySelector('#cup1-image');
+const cup2ImageEl = document.querySelector('#cup2-image');
+const cup3ImageEl = document.querySelector('#cup3-image');
+
+const cup1ButtonEl = document.querySelector('#cup1-button');
+const cup2ButtonEl = document.querySelector('#cup2-button');
+const cup3ButtonEl = document.querySelector('#cup3-button');
+
+const totalGamesSpanEl = document.querySelector('#total-games');
+const totalWinsSpanEl = document.querySelector('#total-wins');
+const totalLossesSpanEl = document.querySelector('#total-losses');
+
+const hardResetButtonEl = document.querySelector('#hard-reset');
+const tryAgainButtonEl = document.querySelector('#try-again');
+
+
 // let state
 
+let totalGames = 0;
+let totalWins = 0;
+
+const cupsPossible = ['cup1', 'cup2', 'cup3'];
+
+tryAgainButtonEl.disabled = true;
+
```

Comment:
Good job keeping the user experience in mind by doing little things like disabling buttons that should not be clickable.